internal/webhooks/vspheremachine_test.go

```diff
@@ -190,6 +231,7 @@ func createVSphereMachine(server string, providerID *string, preferredAPIServerC
 					PreferredAPIServerCIDR: preferredAPIServerCIDR,
 					Devices:                []infrav1.NetworkDeviceSpec{},
 				},
+				PciDevices: pciDevices,
```

Comment:
note, the correct PascalCase for this is PCIDevices, but that seems to already in the codebase.